docs/sources/installation/rhel.md

```diff
@@ -47,6 +47,14 @@ running on kernels shipped by the distribution. There are things like namespace
 changes which will cause issues if one decides to step outside that box and run
 non-distro kernel packages.
 
+> **Warning**:
+> Please make sure that your system is up to date by installing updates
+> using `yum update` and rebooting your system.
+> Keeping the system up to date is recommended to ensure your system
+> receives fixes for critical security vulnerabilities and severe bugs.
+> The fixes for severe bugs include fixes for potential kernel panics
```

Comment:
We've had many users running older kernels on older versions of RHEL 6 / CentOS 6 and they were reporting panics and crashes we couldn't reproduce elsewhere.

We've repeated many times on the issues that the RHEL 6 kernel is receiving essential backports and that it should be kept up to date, but we're still receiving bug reports from systems running really old kernels which weren't officially supported with Docker by Red Hat.

This is a problem because Docker itself gets blamed for these crashes, but there's nothing to be done if they're running kernels with bugs which have already been fixed.

Since this 2.6.32 is a custom kernel with a lot of backports and fixes, staying up to date is even more important than updating from 3.18.4 to 3.18.5.

This is why it's very specific.